src/components/Thanks/ThanksPageShare.vue

```diff
@@ -84,6 +84,15 @@
 							Introduce someone new to Kiva and we'll give you $25 to support another borrower. Your Kiva Lending Credit will be applied automatically.
 						</p>
 					</template>
+					<template v-else>
+						<h1	class="thanks__headline-h1 tw-mt-1 tw-mb-3 tw-text-left">
+							<span class="fs-mask">{{ this.lender.firstName }}</span>, can you share this loan with one more person?
+						</h1>
+						<p class="tw-text-h3 tw-m-0 thanks__base-text">
+							<!-- eslint-disable-next-line max-len -->
+							{{ this.loan.name }} only needs {{ calculatePeopleQtyToGoal() }} more people to lend $25 and their loan could be fully funded in a matter of hours!
```

Comment:
@edwinf-barreram Please mask the Borrower Name as well. It is masked in the original ThanksPage below already.

Reply:
Done!